File: app/day04/parts.ml
open Support.Utils

let file = "day00/input.txt"

let parse_line (line: string) = 
  String.split_on_char ' ' line
;;

let _compute (lines: string list) = 
  List.map parse_line lines
;;

let () = 
  let lines = read_lines file in
  let _ = _compute lines in
  print_endline "Not Implemented"
;;

File: app/day05/parts.js
const { readFile, strToInt } = require("../support/utils");

const compute_part_1 = () =>
  readFile("day00/input.txt")
    .split("\n")
    .map((line) => !!line)
    .reduce((acc, val) => (val ? acc + 1 : acc), 0);

const compute_part_2 = () =>
  readFile("day00/input.txt")
    .split("\n")
    .map((line) => !!line)
    .reduce((acc, val) => (val ? acc + 1 : acc), 0);

console.assert(compute_part_1() == 0);
console.assert(compute_part_2() == 0);
